common/advanced-options.coffee

```diff
@@ -8,6 +8,6 @@ exports.group =
 		message: 'Check for updates every X minutes'
 		name: 'appUpdatePollInterval'
 		type: 'number'
-		min: 0
-		default: 1
+		min: 10
```

Comment:
This is going to change the minimum just for configuration UIs, not any of the model or validation elsewhere - do we want to change it everywhere else too?

The [actual schema](https://github.com/resin-io-modules/resin-device-config/blob/master/lib/schema.coffee#L71-L75) for example has no minimum, and [the supervisor itself](https://github.com/resin-io/resin-supervisor/blob/44e5778acc38cf83cdfaa3536cf0eb47bd6d4cde/src/device-config.coffee#L41) defaults to 1 minute. Do we want to change those too? Might break some other things (I think we've got hardcoded defaults in a few places which'll conflict with this), but otoh it would be nice for these to all be consistent.

Reply:
@pimterry See https://github.com/resin-io/resin-api/issues/864 AFAIK we've updated this poll interval to 10 minutes on all devices